slacktee.sh

```diff
@@ -77,7 +92,9 @@ function send_message()
 		found_pattern_prefix=""
 	fi
 
-	escaped_message=$(echo "$textWrapper\n$message\n$textWrapper" | sed 's/"/\\"/g' | sed "s/'/\\'/g" )
+	escaped_message=$(echo "$textWrapper\n$message\n$textWrapper" \
+		| sed 's/"/\\\"/g' \
+		| sed "s/'/\\\'/g")
```

Comment:
I would prefer to keep original code.
As far as tested, new code doesn't work if the input string contains single quotes.
e.g.) `echo "'a'" | ./slacktee.sh`